experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/approval_rejection_controls.html

```diff
@@ -9,18 +9,38 @@
   </div>
 {% endif %}
 <div id="review-controls">
-  <div class="alert alert-primary">
-    <p>
-      <strong>{{ experiment.latest_review_requested_by|add:" requested to" }} {{ experiment.review_messages }}</strong>
-    </p>
-    <button type="button"
-            class="btn btn-success"
-            hx-post="{% url approval_url slug=experiment.slug %}"
-            hx-select="#content"
-            hx-target="#content"
-            hx-swap="outerHTML">Approve and {{ action_label }}</button>
-    <button type="button" class="btn btn-danger" id="reject-button">Reject</button>
-  </div>
+  {% can_review_experiment experiment as can_review %}
+  {% if  not can_review %}
```

Comment:
seems like this could be shortened?
```suggestion
  {% if not can_review_experiment experiment %}
```